tests/execution.rs

```diff
@@ -2751,8 +2751,8 @@ fn test_reloc_64_relative_data() {
         config,
         [],
         (),
-        TestContextObject::new(4),
-        ProgramResult::Ok(ebpf::MM_RODATA_START),
+        TestContextObject::new(3),
+        ProgramResult::Ok(ebpf::MM_BYTECODE_START),
```

Comment:
All these tests are returning the inverted addresses, aren't they?

Reply:
I rebased the PR ontop of the changes which allow us to now swap / rename these two constants.